specs/pluggable_alert_delivery.adoc

```diff
@@ -107,6 +132,7 @@ and for a status based alert:
   },
   'Type': 'status',
   'severity': 'Critical',
+  'significance': 'HIGH',
 }
```

Comment:
No its not 2 different schema even currently. Its just 2 different examples shown above.